doc: nrf: release-note: bootloader & DFU (#28103)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  The new function is defined and maintained in the Zephyr repository, providing a very similar API and functionality.
  The old function is deprecated and kept for backward compatibility, but it will be removed in the future.

> * Added implementation of USB firmware loader. It is possible to by selected for a sysbuild project by enabling the :kconfig:option:`SB_CONFIG_FIRMWARE_LOADER_IMAGE_USB_MCUMGR` Kconfig option.

Reviewer: Can you please rephrase the sentence?

Author: Added implementation of USB firmware loader :ref:fw_loader_usb_mcumgr, which is possible to by selected for a sysbuild project by enabling the :kconfig:option:SB_CONFIG_FIRMWARE_LOADER_IMAGE_USB_MCUMGR Kconfig option.

> * Added implementation of USB firmware loader. It is possible to by selected for a sysbuild project by enabling the :kconfig:option:`SB_CONFIG_FIRMWARE_LOADER_IMAGE_USB_MCUMGR` Kconfig option.

Reviewer 2: ```suggestion
* Added implementation of USB firmware loader.
  You can select it for a sysbuild project by enabling the :kconfig:option:`SB_CONFIG_FIRMWARE_LOADER_IMAGE_USB_MCUMGR` Kconfig option.
```

> * Added support for ``nRF54L15tag`` board target with MCUboot dual bank scheme.

Reviewer: Added support for LM20A and LM20B...
Added experimental support for LS05...
Added possibility to update "Loader" in Single Slot
Single slot and RAM Loader modes are now supported

Author: `Added possibility to update "Loader" in Single Slot` is already there

> * Added experimental support for ``nRF54LM20dongle`` with MCUboot singel slot scheme and USB buttonless DFU support with the firmware loader.

Reviewer 2: ```suggestion
* Added experimental support for the ``nRF54LM20dongle`` board target.
  This target uses the MCUboot single-slot scheme and supports USB buttonless DFU through the firmware loader.
```

> * Glitch detector is enabled when |NSIB| is running on nRF54L15 SoC.

Reviewer 2: ```suggestion
* Enabled the glitch detector when |NSIB| is running on the nRF54L15 SoC.
```
